tests/system/src/test/java/test/robot/javafx/scene/control/behavior/BehaviorRobotTestBase.java

```diff
@@ -320,14 +320,17 @@ protected Runnable shift(KeyCode k) {
     }
 
     /**
-     * Convenience alias for Thread.sleep() that does not throw an exception.
+     * Convenience alias for Thread.sleep() that rethrows InterruptedException
+     * as an AssertionError.
      * @param ms the timout in milliseconds
      */
     protected void sleep(int ms) {
         try {
+            // KCR: this should be sleep(ms) not a hard-coded 1
```

Comment:
good thing it's never called, but it is my fault.
this method was later replaced with `waitForIdle`

I can make this change as part of #1569

Reply:
If it isn't being used, then I like your idea of deleting it. Especially since this seems redundant given `Util.sleep(ms)`. I'll revert my changes and let you delete it as part of #1569.

Reply:
On closer inspection, I don't think this should be removed as part of PR #1569, since that file is already using JUnit 5 and is not touched by your PR.

Either I can remove the method with this PR or we just leave it for a follow-up. Removing it with this PR seems easiest, since I am already modifying that method as part of this PR (along with other methods that catch and ignore InterruptedException from sleep).

Reply:
I agree with removing the method in this PR.  Thanks!

Reply:
Done. Thanks.